make/package-kube

```diff
@@ -9,7 +9,7 @@ ARTIFACT=uaa-kube-${GIT_DESCRIBE}.zip
 
 rm ${GIT_ROOT}/${ARTIFACT} 2>/dev/null || true
 
-cd kube && zip -r9 ${GIT_ROOT}/${ARTIFACT} * \
+cd kube && zip -r9 ${GIT_ROOT}/${ARTIFACT} * ../kube-test/* \
```

Comment:
The resulting archive looks like
```
Archive:  uaa-kube-0.0.1-43-g5ae13ad.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2017-06-09 15:46   bosh/
     6568  2017-06-20 15:42   bosh/uaa.yml
     4104  2017-06-20 15:42   bosh/mysql.yml
        0  2017-06-14 10:43   secrets/
    14153  2017-06-20 15:42   secrets/secret-1.yml
      271  2017-06-10 14:02   ../kube-test/exposed-ports.yml
      149  2017-06-09 11:58   ../kube-test/storage-class-host-path.yml
---------                     -------
    25245                     7 files
```
The paths for the kube-test files do not look nice/correct.

Also note, the `kube-dist` result in the SCF integration branch is not including these files either, and likely should.